deb/build/build.sh

```diff
@@ -13,9 +13,15 @@ mkdir -p $D
 # so that multiple builds can go on concurrently
 cp -R $dir/* $D
 
+# Create a description temp file
+sed -i.bak -e 's/^\s*$/./' -e 's/^/ /' $DESCRIPTION_FILE
```

Comment:
YES.  This is exactly the approach I proposed -- just want to make sure the transformation is right: was this tested?  

I think you need the -r argument to use an extended regex.   Maybe best to provide the transformed output result in the description so we see that it generates correctly.